uaaextras/webapp.py

```diff
@@ -58,13 +58,24 @@
         "socket_connect_timeout": 0.5
     }
 
-# Connect to redis
-try:
-    r = redis.StrictRedis(**redis_env)
-    r.info()
-except Exception as exc:
-    logging.warn('Unable to connect to Redis: {0}'.format(exc))
-    r = None
+
+class RedisWrapper(object):
+    def __init__(self, **kwargs):
+        self.kwargs = kwargs
+
+    @property
+    def redis(self):
+        if not hasattr(self, '_redis'):
```

Comment:
It's not set in the constructor, so until this method successfully connects to redis, `hasattr(self, '_redis')` should be `False`.